### PR TITLE
chore: pin docs and examples react dep to 17

### PR DESCRIPTION
**Pull request by reesscot**
#### Description of changes
This PR updates our react dependency in docs and next-example to 17

#### Checklist

- [x] PR description included

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by joebuono — APPROVED** [review]
LGTM

**Review by 0618 — APPROVED** [review]

**Review by jacoblogan — APPROVED** [review]

File: docs/package.json [hunk 1]
```diff
@@ -38,9 +38,9 @@
     "next-sitemap": "^1.6.203",
     "prism-react-renderer": "1.2.1",
     "raw-loader": "^4.0.2",
-    "react": "latest",
+    "react": "17",
```
Comment (0618) [inline docs/package.json:41]: should we use `^17.0.0`?

Comment (reesscot) [reply docs/package.json:41]: Isn't that pretty much equivalent to `17`?

Comment (0618) [reply docs/package.json:41]: I need to look it up 😂

Comment (reesscot) [reply docs/package.json:41]: Probably wouldn't actually ever want `17.0.0` anyway. With `17` we should always get the latest version of 17, which is what we want for docs.

**Review by ErikCH — APPROVED** [review]
👍

File: docs/package.json (hunk 1 above)
Comment (0618) [reply docs/package.json:41]: Cool! Verified here https://semver.npmjs.com/ 😁

Comment (ErikCH) [inline docs/package.json:41]: Maybe ^17 instead?

**Pull request merged**